plugins/com.google.cloud.tools.eclipse.sdk.test/src/com/google/cloud/tools/eclipse/sdk/CloudSdkManagerTest.java

```diff
@@ -17,13 +17,26 @@
 package com.google.cloud.tools.eclipse.sdk;
 
 import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
 
+import org.junit.After;
 import org.junit.Test;
 
 public class CloudSdkManagerTest {
 
+  @After
+  public void tearDown() {
+    CloudSdkManager.forceManagedSdkFeature = false;
+  }
+
   @Test
   public void testManagedSdkOption() {
-    assertFalse(CloudSdkManager.isManaged());
+    assertFalse(CloudSdkManager.managedSdkFeatureEnabled());
```

Comment:
I think an `is` prefix helps understand the purpose — I had to reread it a couple of times as I imagined that it enabled the management feature and returned the previous state.